lib/models/wallet_connect.dart

```diff
@@ -0,0 +1,25 @@
+/// WalletConnect Methods
+abstract class WcMethods {
+  /// Get the client to request a list of all accounts in wallet
+  static const wRequestAccounts = "wallet_requestAccounts";
+
+  /// Send qubic from an account to a destination address
+  static const wSendQubic = "sendQubic";
+
+  /// Send assets from an account to a destination address
+  static const wSendAsset = "sendAsset";
+}
+
+abstract class WcEvents {
+  /// The list of accounts has changed
+  static const accountsChanged = "accountsChanged";
+
```

Comment:
Based on what I was reading about WC, the dApp is generally expected to handle balance updates independently, without relying on a WC event.
Triggering events for balance changes would cause unnecessary updates and traffic between the wallet and the dApp.
I don't think "tokenAmountChanged" and "amountChanged" are needed/recommended.

Reply:
Yes, we can do away with this, it's just for dev convenience. Traffic is not so big of a deal (some kb when amounts are changed) but for sure we can hide / disable this functionality. Just guide me on how you would like to proceed

Reply:
We will leave it by now and let the dapps decide to use it or not.